subxt/src/utils/multi_address.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+//! The "default" Substrate/Polkadot Address type. This is used in codegen, as well as signing related bits.
+//! This doesn't contain much functionality itself, but is easy to convert to/from an `sp_runtime::MultiAddress`
+//! for instance, to gain functionality without forcing a dependency on Substrate crates here.
+
+use codec::{
+    Decode,
+    Encode,
+};
+
+/// A multi-format address wrapper for on-chain accounts. This is a simplified version ob Substrate's
+/// `sp_runtime::MultiAddress`. To obtain more functionality, convert this into that type.
```

Comment:
perhaps explain than `#[cfg(feature = "substrate-compat")]` is required for converting it `sp_runtime::MultiAddress`

Reply:
When I wrote this I was assuming that people could manually convert it if they wanted (because it has the same structure so is easy to see how), and then I added some From things behind `substrate-compat` becasue I needed them in the PairSigner impl to keep things simple!

Def worth mentioning substrate-compat here now though yup :)